src/test/java/com/redhat/cloud/notifications/TestLifecycleManager.java

```diff
@@ -54,6 +56,7 @@ private void configureObjectMapper() {
         ObjectMapper mapper = DatabindCodec.mapper();
         mapper.setFilterProvider(filterProvider);
         mapper.registerModule(new JavaTimeModule());
+        mapper.configure(FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Do we have extra properties in some cases? I think we should clean them if possible. Not really important for the internal case, but we should try to ensure what we are returning in the rest APIs.